Recursion/Print_Decrease_Increase_of_a_number.cpp

```diff
@@ -0,0 +1,51 @@
+// Write a program that will print a number first in decrease order then in increase order using recursion.
```

Comment:
We are not instructing anyone to write a program using the provided description.

```
// The program prints a number first in decreasing order and then in increasing order using recursion.

```